framework/source/class/qx/test/log/Filters.js

```diff
@@ -0,0 +1,64 @@
+/* ************************************************************************
+
+   qooxdoo - the new era of web development
+
+   http://qooxdoo.org
+
+   Copyright:
+     2004-2009 1&1 Internet AG, Germany, http://www.1und1.de
+
+   License:
+     LGPL: http://www.gnu.org/licenses/lgpl.html
+     EPL: http://www.eclipse.org/org/documents/epl-v10.php
+     See the LICENSE file in the project's top-level directory for details.
+
+   Authors:
+ * Fabian Jakobs (fjakobs)
+
```

Comment:
author is me :)

I'm changing the Filter to use the MIT license, does it make sense to do the same on files we edit also, ie qx.log.Logger?